# Comparative genomic analysis of glutamate transporters reveals the evolutionary landscape and splicing plasticity of truncated isoforms

Abstract

Glutamate transporters are fundamental in maintaining excitatory neurotransmitter homeostasis within the central nervous system, where their dysfunction is implicated in numerous neurological disorders. These transporters typically exhibit significant splicing diversity, producing multiple truncated isoforms through alternative splicing mechanisms. This study presents a comprehensive analysis of the evolutionary landscape and splicing plasticity of the protein-coding isoforms, utilizing comparative genomics, evolutionary statistics, and structural analyses. Our analyses indicates that the RNA transcripts generated by genes may affect the outcome of specific selection models and thus contribute to the diversification of the glutamate transporter subfamily. Neutral evolution allows for genetic variability without immediate functional consequences and serves as a reservoir for potential future adaptation. This is typically evident in genes of EAA3 and EAA5, which exhibit a substantial reduction in protein-coding isoforms in hominid species. Evolutionary patterns suggesting stable, optimized states (positive or neutral Tajima's D), reducing pressure for isoform diversity. Despite the variations following a more neutral evolution model, the secondary structures of EAAT RNAs are selectively conserved amongst hominids. In contrast, evidence of selection-driven changes is apparent in the pairwise genetic variation profiles of the EAA1 and EAA2 cDNAs, with substantially more negative Tajimaâ€™s D values. Translated isoform protein structures that mimic canonical helices were also conserved. Therefore, this may reflect an evolutionary pressure to preserve protein structures relative to RNA structures. By characterizing the conserved features of RNA splicing, our models provide a basis for the design of targeted therapeutic interventions for neurological disorders, exploiting the evolutionary optimization inherent in RNA splicing processes.

Key Words: Truncated isoforms, Membrane proteins, Alternative splicing, Comparative Genomics, RNA evolution
